modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -0,0 +1,1138 @@
+
+In {productname} 6.4.1, it is now not possible to highlight inline boundary elements that are contained inside a `contenteditable="false"` element.
+
+=== Clicking on a disabled split button would call the onAction callback
+//#TINY-9504
+
+A bug was found in versions of {productname} prior to the 6.4.1 release, that would call the `onAction` callback for *split buttons* regardless of the action being `disabled`. This meant it was possible for the user to call `onAction` unintentionally
+
+As a fix to the issue, {productname} 6.4.1 introduced a new conditional check to identify *if* the button is `disabled`  before calling the `onAction` callback.
+
+In {productname} 6.4.1, clicking on a disabled split button won't call the `onAction` callback anymore.
+
+=== The "Edit Link" dialog incorrectly retrieved the URL value when opened immediately after the link insertion
+//#TINY-7993
+
+A bug was found within the context menu and dialog when inserting `links` in previous versions of {productname}. The issue resulted in the `open link` being deactivated within the context menu and dialog when the user would execute a specific chain of clicks. Such as:
+
+.Example
+====
+. Opening the editor
+. Type “test1 test2 test3” into the editor
+. Double click to select “test2”
+. Use the context menu to insert a new link over the text “test2”
+. Add a valid URL
+. Press `Enter`
+. Without taking any other action, right click on “test2”
+
+In this example the bug would deactivate the `open link` button within the context menu and dialog, which removes the users ability to reopen the “open link” dialog.
+====
+
+The fix now allows all three link-plugins (Link, Remove link and Open link) to work within the context menu items.
+
+In {productname} 6.4.1, when the user executes right `mouse click` on the link selection immediately after attaching a link to a `<string>`, the link-plugins now work as expected.
+
+=== Editor commands ForwardDelete and Delete were deleting contents inside noneditable elements
+//#TINY-9477
+
+Previously, when using the `ForwardDelete` or `Delete` commands in earlier versions of {productname}, the contents within `contenteditable="false"` elements were deleted. This meant that if there were nested elements within a `contenteditable="false"` root element that were also set to `contenteditable="false"`, they could be deleted from the root element.
+
+To resolve this issue, {productname} 6.4.1 included several enhancements to prevent all commands and keyboard functions, including delete/forwarddelete, from affecting `contenteditable="false"` content.
+
+Consequently, in {productname} 6.4.1, using the forwarddelete or delete keys on `contenteditable="false"` elements within a `contenteditable="false"` element will no longer delete the contents within the `contenteditable="false"` elements.
+
+=== Backspace or delete keys was deleting contents inside noneditable elements
```

Comment:
I really don't think this needed two sections, but I'm not in a position to combine them myself today